.eslint-resolver.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+const assign = require ( 'lodash/assign' );
+const debugFactory = require( 'debug' );
+const nodeResolver = require( 'eslint-import-resolver-node' );
+const path = require( 'path' );
+const webpackResolver = require( 'eslint-import-resolver-webpack' );
+
+/**
+ * Internal dependencies
+ */
+const webpackResolveConfig = require( './webpack.config.resolve' );
+
+/**
+ * Module variables
+ */
+const log = debugFactory('eslint-plugin-import:resolver:wp-calypso-resolver');
+
+const nodeResolverConfig = {
```

Comment:
Sure - the reason why we need separate resolving configs is due to how files across our project assume a different root directory. For client files, this includes `/client/`
and `/client/extensions/`; for server files, this includes `/`, `/client/`, and `/server/`. Test files tend to add `/test/` as a root directory as well.

I do think it should be pulling from `webpack.config.node.js` for server-side module importing. I'll push out a revision to address this shortly.